meson.build

```diff
@@ -244,6 +244,7 @@ if meson.version().version_compare('>= 0.53')
   }, section: 'Other Inputs')
 
   summary({
+    'Background music': true,
```

Comment:
Capitalize "Music" like in configure.ac.

Reply:
Fixed as part of [this commit](https://github.com/audacious-media-player/audacious-plugins/commit/ff69402b28597d41ac73c29e04ec9dbfd79d5a8f)